packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -416,6 +418,16 @@ class FloatingActionButton extends StatelessWidget {
   ///  * [MaterialTapTargetSize], for a description of how this affects tap targets.
   final MaterialTapTargetSize? materialTapTargetSize;
 
+  /// Whether detected gestures should provide acoustic and/or haptic feedback.
+  ///
+  /// For example, on Android a tap will produce a clicking sound and a
+  /// long-press will produce a short vibration, when feedback is enabled.
+  ///
```

Comment:
We should add mention here that when null, the theme will be used, and if that is null what the default behavior is.

Reply:
Sure I'll update the docs.

Reply:
@Piinks, its done :)